js/client-teambuilder.js

```diff
@@ -983,11 +983,14 @@
 				if (exports.BattleFormats) {
 					buf += '<li class="format-select">';
 					buf += '<label class="label">Format:</label><button class="select formatselect teambuilderformatselect" name="format" value="' + this.curTeam.format + '">' + (Tools.escapeFormat(this.curTeam.format) || '<em>Select a format</em>') + '</button>';
-					buf += ' <button name="validate" class="button"><i class="fa fa-check"></i> Validate</button></li>';
 				}
+
 				if (!this.curSetList.length) {
 					buf += '<li><em>you have no pokemon lol</em></li>';
+				} else {
+					buf += '<button name="validate" class="button"><i class="fa fa-check"></i> Validate</button></li>';
```

Comment:
Really? I swear we have styling for disabled buttons.

Try the CSS class `disabled`.